javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,18 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === "") {
+        console.log("Please enter a valid student name");
+    }else if (class07Students.includes(studentName)) {
+        console.log(`Student ${studentName} is already in the class`)
+    }else if(studentName === "Queen") {
+    class07Students.push(studentName);
+  }else if (class07Students.length === 6 && studentName != "Queen") {
```

Comment:
This works fine. As an idea, you already made a function to calculate the number of students. What do you think of reusing it here?